src/server/database/migrations/20170714185201-removeEmptyContent.js

```diff
@@ -0,0 +1,10 @@
+exports.up = async (r) => {
+  r.table('Project')
+    .filter((doc) => doc('content').match('"blocks":\\[]'))
```

Comment:
oh no worries on that guy, that string is generated via JSON.stringify so it conforms to the exact spaceless, double quoted JSON standard